src/components/Pools/PoolList/PoolListV2.tsx

```diff
@@ -60,7 +60,10 @@ const PoolListV2: React.FC<EarnProps> = ({ version, stakingInfos, setMenu, activ
         .filter((element) => !!element);
       setStakingInfoData(sortedFarms);
     }
+  };
 
+  useEffect(() => {
+    sort();
```

Comment:
1. I am not able to understand what was the issue here. can you please tell me what is issue and how its fixed.
2. Also can you sort by APR or Liq then search something and check whether its sorted or not ?

Reply:
1. this problem https://app.clickup.com/t/2tujnh2
2. Nop, there are still some bugs, I will fix
[Gravação de tela de 27-10-2022 09:24:36.webm](https://user-images.githubusercontent.com/37405304/198297470-37bf1e33-1e42-4ad1-8f70-2beba9c90693.webm)